### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by jglev** [jglev/obsidian-apply-patterns-plugin#35] [Announcements]
A significant release! :tada:

- The Settings menu has been completely redesigned, and should now be more mobile-friendly.
- The position of the cursor, or selection of text, after applying a pattern is now able to be customized from the Settings menu.
- Links in the Readme have been updated.
- Patterns and Rules now ask for confirmation before being deleted.

Thank you to @claremacrae and @feliperearden for beta testing earlier versions of this release!

<hr /><em>This discussion was created from the release <a href='https://github.com/jglev/obsidian-apply-patterns-plugin/releases/tag/1.4.0'>1.4.0</a>.</em>

**Comment by FelipeRearden** [comment]
> Thank you to @claremacrae and @FelipeRearden for beta testing earlier versions of this release!

It's always a honor to talk with you @jglev !!! You can count on me for more beta releases in the future :)
- Apply Patterns and Carry-Forward are the most important plug-ins for almost all of my workflows. I'm very happy with this new release. 

I hope you could have some time in the future to make our beloved Carry-Forward available for Obsidian Mobile as well 🙏 🙏 🙏 

Have a great day!!!!!!!

**Comment by claremacrae** [comment]
I was happy to do the testing. Thanks for the really helpful plugin!
